### ISSUE TITLE
After some steps, all the NNs always output same action

### ISSUE THREAD

**Issue by Eify666666** [ikostrikov/pytorch-a3c#75]
I'm training a A3C these days, but the NN always take the same action, after some steps.
The game I train for is similar to playing Go. There will be few reward in the short term. So it hard to learn something useful for the NN form the game. Maybe that is where the problem is. I tried '    torch.nn.utils.clip_grad_norm(lnet.parameters(), 50) ', and used relu as activate function. But it doesn't work.

**Comment by RuoyuG** [comment]
I meet same problem, it looks like stuck in a local optimal. Do you solve it?
